ee/tabby-db/migrations/07-smtp-credentials/up.sql

```diff
@@ -0,0 +1,5 @@
+CREATE TABLE smtp_info(
+    email VARCHAR(255),
+    password VARCHAR(255),
+    mailserver_url VARCHAR(255),
```

Comment:
based on https://crates.io/crates/lettre, it seems the field we need are:
1. smtp_username
2. smtp_password
3. smtp_server